minetester/scripts/test_loop_parallel.py

```diff
@@ -56,16 +57,16 @@ def _init():
     xserver = start_xserver(x_display)
 
     # Start loop
-    render = True
+    render = False
     obs = venv.reset()
     done = [False] * num_envs
     step = 0
     while step < max_steps:
         print(f"Elapsed steps: {venv.get_attr('_elapsed_steps')}")
```

Comment:
I'm getting an error on this line when running the parallel test loop It's telling me get_attr doesn't exist for the AsyncVectorEnv

Reply:
requires gym>=0.23.1